nexus/types/src/deployment.rs

```diff
@@ -926,25 +951,71 @@ pub enum BlueprintPhysicalDiskDisposition {
     InService,
 
     /// The physical disk is permanently gone.
-    Expunged,
+    Expunged {
+        /// Generation of the parent config in which this disk became expunged.
+        as_of_generation: Generation,
+
+        /// True if Reconfiguration knows that this disk has been expunged.
+        ///
+        /// In the current implementation, this means:
```

Comment:
Nit - maybe

```suggestion
        /// In the current implementation, this means either:
```

Reply:
Done in 55a65f8d01ed8e54a7ade5b84eeae83a937956dd